README.md

```diff
@@ -159,6 +161,38 @@ Report specific arguments:
  * `--output-format` - output format: text or json
  * `--cassandra` - Enable advanced cql queries for Apache Cassandra database
 
+
+### Distributed storages
+
+Gemini supports different distributed storages in local and cluster mode. It already includes all necessary jars as a part of fat jar.
+
+#### HDFS
+
+Path format to git repositories: `hdfs://hdfs-namenode/path`
+
+To configure HDFS in local or cluster mode please consult [Hadoop documentation](https://hadoop.apache.org/docs/r2.7.0/index.html).
```

Comment:
I noticed that [here](https://github.com/src-d/gemini/blob/master/project/Dependencies.scala#L10) we're using `v2.7.2` of the client, does the client follow the same versioning? In that case we should point to the corresponding doc.